src/main/DAL/MetaQuestion/MetaQuestionRepository.js

```diff
@@ -84,6 +181,10 @@ class MetaQuestionRepository {
                     model: this.#Keyword,
                     as: 'keywords',
                 },
+                {
+                    model: this.#Appendix,
+                    as: 'appendix',
```

Comment:
required: false,

Reply:
if the MQ is not connected to an Appendix this field will be `null`. as defined by the relationship.

Reply:
we dont want to work with null.

you so what it did to us...

make sure it is undefined

Reply:
This is too much logic for DAL. 
I would actually like to get a good amount of information (even maybe a little extra) from DAL layer and then Business layer can manipulate it in whatever form we want. that way DAL layer won't need many refactors.